header/log_includes/compress_log.h

```diff
@@ -0,0 +1,4 @@
+#include<string>
+using namespace std;
+
```

Comment:
Add doxygen comments before function prototype.
https://en.wikipedia.org/wiki/Doxygen
Example:
      /**
       * a normal member taking two arguments and returning an integer value.
       * @param a an integer argument.
       * @param s a constant character pointer.
       * @return The test results
       */
       int testMe(int a,const char *s);